testing/jest/widget.tests.tsx

```diff
@@ -86,28 +86,28 @@ describe('Widget', () => {
     describe('width/height', () => {
       it('should have the ability to be a function', () => {
         const widget = render({ width: () => 50, height: () => 'auto' });
-        const props = widget.props() as WidgetProps;
+        const props = widget.props() as any;
```

Comment:
It seems like we should not use ts inside tests 😃

Reply:
`widget.props()` doesn't return only `WidgetProps` type. For example, we have no `style` property in `Widget` component, but `widget.props()` has this property.

We can write next:
```tsx
const props = widget.props() as WidgetProps && { style: Object };
```